src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
Link to the stylesheet should have an extension of `.css`, not `.scss`. `.scss` files are preprocessed by Sass into `.css` files, which are the ones browsers can understand.